src/main/java/com/notably/commons/core/path/AbsolutePath.java

```diff
@@ -1,53 +1,123 @@
 package com.notably.commons.core.path;
 
+import java.util.ArrayList;
 import java.util.List;
+import java.util.Objects;
 
 import com.notably.commons.core.path.exceptions.InvalidPathException;
 
 /**
  * TODO: Add Javadoc
  */
 public class AbsolutePath implements Path {
+    public static final String INVALID_ABSOLUTE_PATH = "Invalid absolute path";
+    public static final String VALIDATION_REGEX = "(\\/(\\p{Alnum}+))+\\/?";
+    private final List<String> paths;
```

Comment:
Oh yes, let's rename this variable to `components`? So that the method `getComponents` feels more natural.